File: experiment_jp_test/j2pyraw/if__0__FIND_MAXIMUM_PRODUCT_OF_A_TRIPLET_IN_ARRAY_1.java
class X { static int maxProduct ( int arr [ ] , int n ) { Arrays . sort ( arr ) ; return Math . max ( arr [ 0 ] * arr [ 1 ] * arr [ n - 1 ] , arr [ n - 1 ] * arr [ n - 2 ] * arr [ n - 3 ] ) ; }
 }

File: experiment/java-python/test/j2pyrawpy/if__0__FIND_MAXIMUM_PRODUCT_OF_A_TRIPLET_IN_ARRAY_1.py
#!/usr/bin/env python
""" generated source for module if__0__FIND_MAXIMUM_PRODUCT_OF_A_TRIPLET_IN_ARRAY_1 """
class X(object):
    """ generated source for class X """
    @classmethod
    def maxProduct(cls, arr, n):
        """ generated source for method maxProduct """
        Arrays.sort(arr)
        return Math.max(arr[0] * arr[1] * arr[n - 1], arr[n - 1] * arr[n - 2] * arr[n - 3])

